src/libra_py/pdos.py

```diff
@@ -145,26 +157,48 @@ def QE_pdos(prefix, emin, emax, de, projections, Ef, outfile, do_convolve, de_ne
             for symb in ang_mom:  # for given angular momentum labels
                 for wfc in range(0,5): # Specify max wfc type index - usually no more than 3, 5 - should be more than enough  
                     for Elt in elements: # for given atom names
-                        filename = prefix+str(a)+"("+Elt+")_wfc#"+str(wfc)+"("+symb+")"  # file
-
-                        if os.path.exists(filename):
 
-                            fa = open(filename,"r")
-                            B = fa.readlines()
-                            check = B[0].split()
-                            fa.close()
+                        if nspin == 4:
+                            K = [0.5, 1.5, 2.5]
+                            for k in K:

```

Comment:
you probably don't need to define the variable K, unless you use it somewhere else. Also, what are those numbers anyway? Please, leave some comments in the code